metricbeat/module/kubernetes/kubeproxy/_meta/data.json

```diff
@@ -0,0 +1,90 @@
+{
+  "@timestamp": "2019-06-01T08:08:00.885Z",
+  "@metadata": {
+    "beat": "metricbeat",
+    "type": "_doc",
+    "version": "8.0.0"
+  },
+  "ecs": {
+    "version": "1.0.0"
+  },
+  "host": {
+    "hostname": "pl51",
+    "name": "pl51",
+    "architecture": "x86_64",
+    "os": {
+      "name": "Ubuntu",
+      "kernel": "4.15.0-48-generic",
+      "codename": "bionic",
+      "platform": "ubuntu",
+      "version": "18.04.2 LTS (Bionic Beaver)",
+      "family": "debian"
+    },
+    "id": "07ed8f34c84f4e6aa92e212a931f2efa",
+    "containerized": false
+  },
+  "service": {
+    "address": "192.168.205.12:10249",
+    "type": "kubernetes"
+  },
+  "kubernetes": {
+    "kubeproxy": {
+      "process": {
+        "cpu": {
+          "sec": 199
+        },
+        "memory": {
+          "resident": {
+            "bytes": 3.4369536e+07
+          },
+          "virtual": {
+            "bytes": 1.42323712e+08
+          }
+        }
+      },
+      "sync": {
+        "rules": {
+          "latency": {
+            "us": {
+              "bucket": {
+                "128000": 4857,
```

Comment:
not for this PR, but the more I see this the more I think we should move towards having bucket (`le`) as a label in events, it would probably simplify queries.

Reply:
agree, let's build that proposal from UI (kibana) downwards.